docs/HowTo/Configure/Configure-TLS.md

```diff
@@ -0,0 +1,113 @@
+description: Configure TLS
+<!--- END of page meta data -->
+
+# Configure TLS
+
+Hyperledger Besu supports TLS for client and server communication. For example, you can
+[configure TLS](../../Concepts/TLS.md) for communication between EthSigner and Besu, and Besu and Orion.
+
+Configure TLS communication from the command line.
+
+**Prerequisites**:
```

Comment:
Similar to comment on EthSigner PR - I think it's worth making it clearer these prereqs are common to client and server.  The title could be changed to reflect this: 
_Prerequisites for Client and Server TLS_

Reply:
Done